docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/delta-lake/page.mdx

---
title: 'Delta Lake'
description: 'Ingest data from Delta Lake tables into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# Delta Lake

Delta Lake is an open-source storage layer that adds ACID transactions and reliability to data lakes. It enables batch and streaming data processing with schema enforcement and versioning.

## Using Delta Lake with GlassFlow

GlassFlow can ingest data from Delta Lake tables into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, joins, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** Delta Lake connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** Delta Lake connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
